polaris-react/src/tokens/token-groups/beta-tokens.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We've aliased keyframe token values from `keyframes.json` to custom css properties to avoid the css custom modules hashing them by default.

<img width="617" alt="Screen Shot 2022-04-06 at 10 00 36 AM" src="https://user-images.githubusercontent.com/21976492/162064842-7f3d6c4b-b8d9-4f6d-b3a5-be6dac634f7a.png">


The other alternative would be to wrap any `animations` consuming the keyframes tokens in [:global{}](https://github.com/Shopify/polaris/pull/3032/files#diff-6540ed5581ce15b789159dfc22825c3f18d22445dd7f999f77e01316b83a4492) which felt gross. 

We've put these values into a `beta-tokens` file instead of `motion` since we want to indicate to public facing consumers that these are experimental values for now and are likely to change. We thought the naming convention felt in line with `legacy-tokens`.